module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -969,7 +969,19 @@ public function getRecommendationSettings($handler = null)
     public function getSearchClassId()
     {
         // Parse identifier out of class name of format VuFind\Search\[id]\Options:
-        $class = explode('\\', get_class($this));
+        $className = get_class($this);
+        $class = explode('\\', $className);
+
+        // Special case: if there's only one part, we may be testing with a mock
+        // object; if so, that's okay, but anything else is unexpected.
+        if (count($class) < 3) {
```

Comment:
Why `< 3` and not `< 4` as the comment at the top suggests?

Reply:
Good point -- I'll change it when I have time to make revisions.